graphene/src/layers/style/mod.rs

```diff
@@ -83,16 +98,23 @@ impl PathStyle {
 	pub fn clear_stroke(&mut self) {
 		self.stroke = None;
 	}
-	pub fn render(&self) -> String {
+
+	pub fn render(&self, mode: (ViewMode, bool)) -> String {
+		// change stroke rendering so solid paths don't dissapear
```

Comment:
Is this a `// TODO;` comment, or is this describing what this function does, or what a part of this function does?

Reply:
I think that was a ```// TODO: ``` from before my time, I'm not quite sure what it means actually.